README.md

```diff
@@ -211,15 +211,12 @@ Other combinations might also work but have not been tested.
 
 | Release | SpringBoot |
 |---------|------------|
-| 7.0.0   | 2.7.16     |
-| 6.3.2   | 2.7.14     |
-| 6.2.2   | 2.7.12     |
-| 6.1.1   | 2.7.10     |
+| 9.0.0   | 3.3.2     |
 
 # 📨Contact
 
-If you have any questions or ideas feel free to create an [issue](https://github.com/kadai-iokadai/issues) or contact us
-via [GitHub Discussions](https://github.com/orgs/kadai-iodiscussions).
+If you have any questions or ideas feel free to create an [issue](https://github.com/kadai-io/kadai/issues) or contact us
```

Comment:
There are many more occurrences of this error, I guess:

![grafik](https://github.com/user-attachments/assets/13c302d7-7867-43b3-b9ce-5c2504e26f84)

Reply:
You are right, I will change them